bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -156,6 +160,11 @@ pub mod pallet {
 		}
 	}
 
+	/// The current operating mode of the pallet.
+	#[pallet::storage]
+	#[pallet::getter(fn operating_mode)]
+	pub type OperatingMode<T: Config> = StorageValue<_, BasicOperatingMode, ValueQuery>;
+
```

Comment:
I think the naming is too vague. Operating mode for what?

```suggestion
	/// The current operating mode for exporting to Ethereum.
	#[pallet::storage]
	#[pallet::getter(fn export_operating_mode)]
	pub type ExportOperatingMode<T: Config> = StorageValue<_, BasicOperatingMode, ValueQuery>;
```